shapely/tests/test_constructive.py

```diff
@@ -881,26 +881,23 @@ def test_oriented_envelope_all_types(geometry):
     assert actual is None
 
 
-@pytest.mark.skipif(shapely.geos_version < (3, 12, 0), reason="GEOS < 3.12")
 @pytest.mark.parametrize(
     "func", [shapely.oriented_envelope, shapely.minimum_rotated_rectangle]
 )
 @pytest.mark.parametrize(
     "geometry, expected",
     [
         (
-            MultiPoint([(0, 0), (10, 0), (10, 10)]),
-            Polygon([[10.0, 10.0], [10.0, 0.0], [0.0, 0.0], [0.0, 10.0], [10.0, 10.0]]),
+            MultiPoint([(1.0, 1.0), (1.0, 5.0), (3.0, 6.0), (4.0, 2.0), (5.0, 5.0)]),
+            Polygon([(1.0, 1.0), (1.0, 6.0), (5.0, 6.0), (5.0, 1.0), (1.0, 1.0)]),
```

Comment:
I changed the example here to be a more distinctive one. With this new example, there is a clear differentce in the "minimum area" and "minimum rectangle" solution. While with the previous input geometry, they were the same and there was actually two possible solutions with the same minimum area (the GEOS version vs our version then also returned a different one of those, I think only because of floating point differences or order of iteration)